privacyidea/api/policy.py

```diff
@@ -175,12 +176,7 @@ def set_policy_api(name=None):
     """
     res = {}
     param = request.all_data
-    if not re.match('^[a-zA-Z0-9_.]*$', name):
-        raise ParameterError(_("The name of the policy may only contain "
-                               "the characters a-zA-Z0-9_."))
-
-    if name.lower() == "check":
-        raise ParameterError(_("T'check' is an invalid policy name."))
+    check_policy_name(name)
```

Comment:
shouldn't we put this in the lib-layer?

Reply:
It has not been in the lib layer the whole time.
The documentation points out, that it is actually a cool thing, that you *can* create policies with other names in the lib layer.

We used the possibility to create other-named policies in the migration scripts:
We created policies with "not-allowed" characters in the migration script. This way we could be sure, that those policies do not exist.

Reply:
I want to keep it at this place.